src/Microsoft.AspNet.Authentication/project.json

```diff
@@ -5,6 +5,7 @@
         "Microsoft.AspNet.DataProtection": "1.0.0-*",
         "Microsoft.AspNet.Http": "1.0.0-*",
         "Microsoft.AspNet.Http.Extensions": "1.0.0-*",
+        "Microsoft.Framework.Caching.Abstractions": "1.0.0-*",
```

Comment:
I will. But honestly, depending on the caching packages was not really a big deal :smile: